source/basic.tex

```diff
@@ -6381,12 +6382,12 @@
 An implementation shall not predefine the \tcode{main} function.
 Its type shall have \Cpp{} language linkage
 and it shall have a declared return type of type
-\tcode{int}, but otherwise its type is \impldef{parameters to \tcode{main}}.
+\keyword{int}, but otherwise its type is \impldef{parameters to \tcode{main}}.
 \indextext{\idxcode{main} function!implementation-defined parameters to}%
 An implementation shall allow both
 \begin{itemize}
-\item a function of \tcode{()} returning \tcode{int} and
-\item a function of \tcode{(int}, pointer to pointer to \tcode{char)} returning \tcode{int}
+\item a function of \tcode{()} returning \keyword{int} and
+\item a function of \tcode{(\keyword{int}}, pointer to pointer to \keyword{char}) returning \keyword{int}
```

Comment:
Missing `\tcode` around the closing `)`.

Reply:
fixed